woocommerce/woocommerce-android
# [Mobile Payments] Handle a payment plugin not installed with API call

Closes: #9682


### Description

The PR adds a button the uses API to install WCPay plugin

As the PR contains the extraction of the ViewState and Events to separate files it'll be easier to review it going with commit by commit

### Testing instructions

* Remove WCPay plugin from the store
* Open More -> Payments -> Error at the botton
* Notice "install" button
* Click it and notice that the plugin is install

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/1fd99323-07fc-40d3-95c6-988209279f36


// cc @iamgabrielma
